abba/json/src/Json.cpp

```diff
@@ -0,0 +1,28 @@
+#include "Json.hpp"
+
+Json::Json(std::string file_name)
+{
+	std::cout << "config filename: " << file_name << std::endl;
+	_file.open(file_name, std::ios::in);
+	if (!_file)
+	{
+		std::cout << "wrong file" << std::endl;
+		exit(1);
```

Comment:
In general, when writing libraries, you really don't want to call exit in case of errors and should prefer to throw either exceptions or return error codes; in the case of a constructor it can only be an exception obviously.
The reason I say this is because something like a parser is likely to be part of a library some other executable would use, which usually wants to define its own behavior for handling errors such as this; a web server for example would never choose to exit and simply to return something like an `InternalServerError` response